app/controllers/apo_controller.rb

```diff
@@ -313,6 +334,13 @@ def redirect
 
   # check that the user can carry out this object modification
   def forbid
+    # TODO: this could be a more granular permission request, see the
+    # Dor::Permissable#can_create_apo? and Dor::Permissable#can_manage_apo? in
+    # https://github.com/sul-dlss/dor-services/blob/permissable/lib/dor/models/permissable.rb
+    #return if current_user.can_manage?(@object, 'apo')
+    #
+    # Note that this is different from User#can_manage? because it
+    # excludes current_user.is_manager
```

Comment:
Disagree with (1).  The link will always exist as long as the branch is not deleted.  The granularity of permission might be important and it is a TODO.  There is a clear suggestion on how the code can function, so (2) is much clearer in the code that in some disassociated ticket or issue that might or might not get closed without acting on it.  I'm not taking out the TODO comments, but feel free to file a ticket/issue and note these TODO lines of code in that issue.  It's likely to add to a long list of role/permission issues that linger in a backlog and don't get resolved.